conduct.md

```diff
@@ -14,11 +14,27 @@ testing fork
 * Communication
 * Participation
 * Conduct
-* Decisions
+## Decision
+
+Decisions should be made in the environment that every team member's opinion
+is considered important. They should be made collectively as a team or at
+least 80% of the majority's consensus. All information needed to make a
+decision will be clearly stated to every team member and all concerns should
+be clearly expressed. We will use data, rational knowledge and specific
+experience to make decisions instead of popular opinion. Team members need to
+objectively analyzed the options for the best of the team. Trust and respect
+the decision made in order to move on the project.
+
+* **Unacceptable (U):**
+ * Making decisions without notifying team members.
```

Comment:
Travis build errors:
conduct.md:29: MD005 Inconsistent indentation for list items at the same level
conduct.md:30: MD005 Inconsistent indentation for list items at the same level

Reply:
Working on it now! Might need to add more.